src/main/java/se/kth/sda/skeleton/user/UserService.java

```diff
@@ -12,14 +13,39 @@
     @Autowired
     private BCryptPasswordEncoder passwordEncoder;
 
-
     public User findUserByEmail(String email) {
         return userRepository.findByEmail(email);
     }
 
+    /**
+     * check if the accountType is valid
+     *
+     * @param user user who's accountType is checked
+     * @return true if accountType is valid, false otherwise
+     */
+    public boolean isAccountTypeValid(User user) {
+
+        return user.getAccountType().equals("human") ||
+                user.getAccountType().equals("pet") ||
+                user.getAccountType().equals("service provider") ||
+                user.getAccountType().equals("caretaker");
+    }
+
+    /**
+     * register a user
+     *
+     * @param user user to be registered
+     * save a user in the userRepository
+     */
```

Comment:
- Capitalise 'R'
- I don't think line 38 is required, but if you want to keep it, please could you move it before `@param`?

```
/**
     * Register a user
     *
     * @param user user to be registered
     */
```

Reply:
Agree.